docs/store/datastore.md

```diff
@@ -175,58 +250,64 @@ profile = DatastoreProfileKafkaTarget(name="test_profile",bootstrap_servers="loc
+- `kwargs_public` &mdash; This is a dictionary (`Dict`) meant to hold a collection of key-value pairs that could represent settings or configurations deemed public. These pairs are subsequently passed as parameters to the underlying `kafka.KafkaConsumer()` constructor. The default value for `kwargs_public` is `None`.
+- `kwargs_private` &mdash; This dictionary (`Dict`) is designed to store key-value pairs, typically representing configurations that are of a private or sensitive nature. These pairs are also passed as parameters to the underlying `kafka.KafkaConsumer()` constructor. It defaults to `None`.
+
+
+```
 profile = DatastoreProfileKafkaSource(name="test_profile",bootstrap_servers="localhost", topic="test_topic")
-register_temporary_client_datastore_profile(profile) or project.register_datastore_profile(profile)
-...
 target = KafkaSource(path="ds://test_profile")
+```
 
-`DatastoreProfileKafkaSource` Class Parameters:
-- *name*: name of the profile
-- *brokers*: This parameter can either be a single string or a list of strings representing the Kafka brokers. Brokers serve as the contact points for clients to connect to the Kafka cluster.
-- *topics*: A string or list of strings that denote the Kafka topics from which data will be sourced or read.
-- *group* (Optional): A string representing the consumer group name. Consumer groups are used in Kafka to allow multiple consumers to coordinate and consume messages from topics. The default consumer group is set to `"serving"`.
-- *initial_offset* (Optional): A string that defines the starting point for the Kafka consumer. It can be set to `"earliest"` to start consuming from the beginning of the topic, or `"latest"` to start consuming new messages only. The default is `"earliest"`.
-- *partitions* (Optional): This can either be a single string or a list of strings representing the specific partitions from which the consumer should read. If not specified, the consumer can read from all partitions.
-- *sasl_user* (Optional): A string representing the username for SASL authentication, if required by the Kafka cluster. It's tagged as private for security reasons.
-- *sasl_pass* (Optional): A string representing the password for SASL authentication, correlating with the `sasl_user`. It's tagged as private for security considerations.
-- *kwargs_public* (Optional): This is a dictionary (`Dict`) that holds a collection of key-value pairs used to represent settings or configurations deemed public. These pairs are subsequently passed as parameters to the underlying `kafka.KafkaProducer()` constructor. It defaults to `None`.
-- *kwargs_private* (Optional): This dictionary (`Dict`) is used to store key-value pairs, typically representing configurations that are of a private or sensitive nature. These pairs are subsequently passed as parameters to the underlying `kafka.KafkaProducer()` constructor. It defaults to `None`.
+`DatastoreProfileKafkaSource` class parameters:
```

Comment:
Can you provide the equal env parameter per variable? @alxtkr77

Reply:
No corresponding env variables